src/components/app/App.js

```diff
@@ -8,13 +8,12 @@ function App() {
   const [housePage, setHousePage] = useState('all')
 
   const onNavBtnClick = house => {
-    console.log('App Click ' + house)
     setHousePage(house)
   }
 
   return (
     <div className="App">
-      <Header />
+      <Header onHeaderTitleClick={onNavBtnClick} />
```

Comment:
So about this naming convention: when passing a function as prop, we tend to call it `onSomething`, when we actually call/invoke the function, we call often it `handleSomething`. 
```suggestion
      <Header onHeaderTitleClick={handleNavBtnClick} />
```

Reply:
Yeah, I changed it to: 
**navBtnClickHandler**